hooks/post_gen_project.py

```diff
@@ -0,0 +1,41 @@
+import re
+import sys
+from pathlib import Path
+
+
+# >>> Optionally clearing comments comments
+COMMENTS_STRUCTURE = {
+    "./.neuro/live.yml": r"(\s*#.*)",
+    "./.neuro/project.yml": r"(\s*#.*)",
+}
+PRESERVE_HINTS_VARIANS = {
+    "yes": True,
+    "y": True,
+    "true": True,
+    "no": False,
+    "n": False,
+    "false": False,
+}
+PRESERVE_HINTS_ANSWER = (
+    "{{ cookiecutter['preserve Neuro Flow template hints'] | lower }}"
+)
+if PRESERVE_HINTS_ANSWER not in PRESERVE_HINTS_VARIANS:
+    print(
```

Comment:
maybe raise exception instead?

Reply:
I was also wondering why not just an exception, but it's a requirement of [cookiecutter](https://cookiecutter.readthedocs.io/en/1.7.2/advanced/hooks.html#exit-with-an-appropriate-status), they handle exit codes and raised exception will add a traceback, which we will not able to hide.
the same says their example (however, implicitly :D)